src/reducers/filter.js

```diff
@@ -0,0 +1,33 @@
+
+const filter = (state = 'ALL', action) => {
+  switch(action.type){
+    case 'ALL':
+      return ''
+    
+    case 'Action':
+      return 'Action'
```

Comment:
Hi guys we can cut this down significantly like this:

action:
```
export const changeFilter = category => {
  return {
    type: `CHANGE_FILTER`,
    category
  }
}
```

reducer:
this way we can return exactly what is picked in category to be saved in store and use it to filter books.
```
switch (action.type) {
    case `CHANGE_FILTER`:
      return action.filter
    default:
      return state
  }
```